mozdownload/scraper.py

```diff
@@ -36,7 +36,10 @@
 APPLICATIONS = ('b2g', 'firefox', 'thunderbird')
 
 # Base URL for the path to all builds
-BASE_URL = 'https://ftp.mozilla.org/pub/mozilla.org'
+BASE_URL = 'http://download.cdn.mozilla.net/pub'
+
+# In case CDN fails
+BACKUP_URL = 'https://ftp.mozilla.org/pub/mozilla.org'
```

Comment:
Wonderful. So @Nebelhom please remove the backup logic. We will stay with CDN only, but please make use of HTTPS and not HTTP, which should really be possible.